index.html

```diff
@@ -214,166 +228,156 @@
+  <script type="text/javascript" src="bower_components/angular/angular.min.js"></script>
+  <script type="text/javascript" src="bower_components/angular-sanitize/angular-sanitize.min.js"></script>
+  <script type="text/javascript" src="bower_components/objectpath/lib/ObjectPath.js"></script>
+  <script type="text/javascript" src="bower_components/moment/moment.js"></script>
+  <script type="text/javascript" src="dist/schema-form.min.js"></script>
+  <script type="text/javascript" src="dist/bootstrap-decorator.min.js"></script>
+  <script type="text/javascript" src="https://cdn.jsdelivr.net/jquery.slick/1.4.0/slick.min.js"></script>
+
+  <script>
+  // @license magnet:?xt=urn:btih:d3d9a9a6595521f9666a5e94cc830dab83b65699&dn=expat.txt Expat
+    $(document).ready(function(){
+      $('.slide-a-long').slick({
+        autoplaySpeed: 5000,
+        dots: false,
+        arrows: false,
+        pauseOnHover: false,
+        autoplay: true
       });
-</script>
-
-    <script>
-
-    $(function() {
-      $('a[href*=#]:not([href=#])').click(function() {
-        if (location.pathname.replace(/^\//,'') == this.pathname.replace(/^\//,'') && location.hostname == this.hostname) {
-          var target = $(this.hash);
-          target = target.length ? target : $('[name=' + this.hash.slice(1) +']');
-          if (target.length) {
-            $('html,body').animate({
-              scrollTop: target.offset().top
-            }, 1000);
-            return false;
+    });
+  // @license-end
+  </script>
+
+  <script>
+  // @license magnet:?xt=urn:btih:d3d9a9a6595521f9666a5e94cc830dab83b65699&dn=expat.txt Expat
+      $(function() {
+        $('a[href*=#]:not([href=#])').click(function() {
+          if (location.pathname.replace(/^\//,'') == this.pathname.replace(/^\//,'') && location.hostname == this.hostname) {
+            var target = $(this.hash);
+            target = target.length ? target : $('[name=' + this.hash.slice(1) +']');
+            if (target.length) {
+              $('html,body').animate({
+                scrollTop: target.offset().top
+              }, 1000);
+              return false;
+            }
           }
-        }
+        });
       });
-    });
-</script>
-
-    <script>
-hljs.initHighlightingOnLoad();
-
-/*
-    Actual usage of angular schema form code if you're looking for it.
- */
-angular.module('schemaForm').controller('PageCtrl', function($scope, $http) {
-      $scope.schema = {
-        'type': 'object',
-        'title': 'Comment',
-        'properties': {
-          'name':  {
-            'title': 'Name',
-            'type': 'string'
+  // @license-end
+  </script>
+
+  <script>
+  // @license magnet:?xt=urn:btih:d3d9a9a6595521f9666a5e94cc830dab83b65699&dn=expat.txt Expat
+  //
+  // Actual usage of angular schema form code if you're looking for it.
+  hljs.initHighlightingOnLoad();
+  angular.module('schemaForm').controller('PageCtrl', function($scope, $http) {
```

Comment:
Oh did you use some kind of code formatter? It looks like a lot of empty lines have disappeared, that might be the problem.